code/game/mecha/mecha_modkit.dm

```diff
@@ -0,0 +1,77 @@
+/obj/item/mecha_modkit
+	name = "mecha modification kit"
+	desc = "A kit to modify your mech. This one doesn't do anything."
+	icon = 'icons/obj/module.dmi'
+	icon_state = "harddisk_mini"
+	var/install_time = 15
+
+/obj/item/mecha_modkit/proc/install(var/obj/mecha/mech, var/mob/user)
+	if(user)
+		to_chat(user, "<span class='notice'>You install [src] into [mech].</span>")
+	return TRUE
+
+/obj/item/mecha_modkit/voice
+	name = "mecha voice modification kit : Standard"
+	desc = "This modification kit updates a mech's onboard voice to Standard."
+	var/nominalsound = 'sound/mecha/nominal.ogg'
+	var/zoomsound = 'sound/mecha/imag_enh.ogg'
+	var/critdestrsound = 'sound/mecha/critdestr.ogg'
+	var/weapdestrsound = 'sound/mecha/weapdestr.ogg'
+	var/lowpowersound = 'sound/mecha/lowpower.ogg'
+	var/longactivationsound = 'sound/mecha/nominal.ogg'
+
+/obj/item/mecha_modkit/voice/install(var/obj/mecha/mech, var/mob/living/carbon/user)
+	if(istype(mech, /obj/mecha/combat/reticence) && user)
```

Comment:
This feels slightly snowflakey... especially since it only goes one-way with the clown-mime feud. Can't think of a good effect for trying to silence the HONKER though, but it feels like there should be one.

Reply:
I did misread that, I mistook the honker mech check as a check for putting the honk voice module into a reticence specifically. This change can be ignored in that case.